pkg/render/manager.go

```diff
@@ -306,6 +312,7 @@ func (c *managerComponent) managerDeployment() *appsv1.Deployment {
 				relasticsearch.ContainerDecorate(c.managerContainer(), c.esClusterConfig.ClusterName(), ElasticsearchManagerUserSecret, c.clusterDomain, c.SupportedOSType()),
 				relasticsearch.ContainerDecorate(c.managerEsProxyContainer(), c.esClusterConfig.ClusterName(), ElasticsearchManagerUserSecret, c.clusterDomain, c.SupportedOSType()),
 				c.managerProxyContainer(),
+				c.managerPacketCaptureContainer(),
```

Comment:
I was wondering if we should start adding resources and limits to the manager deployment. I'm not sure if you would be able to come up with good numbers or otherwise could add a comment for this container detailing how much it would roughly require going forward to your best estimate.